TLM/TLM/U/Autosize/UPadding.cs

```diff
@@ -0,0 +1,38 @@
+namespace TrafficManager.U.Autosize {
+    public struct UPadding {
+        public float Top;
+        public float Right;
+        public float Bottom;
+        public float Left;
+
+        public static UPadding Zero() {
+            return new() {
+                Right = 0f,
+                Top = 0f,
+                Left = 0f,
+                Bottom = 0f,
+            };
+        }
+
+        /// <summary>Set padding on all sides to <see cref="UConst.UIPADDING"/> (4f).</summary>
+        public static UPadding Const() {
```

Comment:
is it possible to have a more meaningful name than Const?

Reply:
Changed to 
```csharp
        public static UPadding Default = new UPadding() {
            Top = UConst.UIPADDING, Right = UConst.UIPADDING, Bottom = UConst.UIPADDING,
            Left = UConst.UIPADDING,
        };

        public static UPadding Zero = new UPadding() {
            Top = 0f, Right = 0f, Bottom = 0f, Left = 0f,
        };
```